src/components/ArrowKeyFocusManager.js

```diff
@@ -0,0 +1,139 @@
+import _ from 'underscore';
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+import * as PropTypeUtils from '../libs/PropTypeUtils';
+import CONST from '../CONST';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.oneOfType([
+        PropTypes.func,
+        PropTypes.node,
+    ]).isRequired,
+
+    /** The initial focused index. */
+    initialFocusedIndex: PropTypeUtils.integerPropType,
+
+    /** The number of items in the list that this component is managing focus for. */
+    listLength: PropTypeUtils.wholeNumberPropType,
+
+    /** A callback executed when the focused input changes. */
+    onFocusedIndexChanged: PropTypes.func,
+
+    /** A callback executed when the enter key is pressed. */
+    onEnterKeyPressed: PropTypes.func,
+
+    /** Should the enter key event bubble? */
+    shouldEnterKeyEventBubble: PropTypes.oneOfType([PropTypes.bool, PropTypes.func]),
+};
+
+const defaultProps = {
+    initialFocusedIndex: 0,
+    listLength: 0,
+    onFocusedIndexChanged: () => {},
+    onEnterKeyPressed: undefined,
+    shouldEnterKeyEventBubble: false,
+};
+
+class ArrowKeyFocusManager extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            focusedIndex: props.initialFocusedIndex,
+        };
+        this.shouldEnterKeyEventBubble = this.shouldEnterKeyEventBubble.bind(this);
+    }
+
+    componentDidMount() {
+        const enterConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
+        const arrowUpConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_UP;
+        const arrowDownConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_DOWN;
+
+        if (this.props.onEnterKeyPressed) {
+            this.unsubscribeEnterKey = KeyboardShortcut.subscribe(enterConfig.shortcutKey, () => {
+                this.props.onEnterKeyPressed(this.state.focusedIndex);
+            }, enterConfig.descriptionKey, enterConfig.modifiers, true, this.shouldEnterKeyEventBubble);
+        }
+
+        this.unsubscribeArrowUpKey = KeyboardShortcut.subscribe(arrowUpConfig.shortcutKey, () => {
+            if (this.props.listLength <= 1) {
+                return;
+            }
+
+            this.setState(
+                (prevState) => {
+                    let newFocusedIndex = prevState.focusedIndex - 1;
+
+                    // Wrap around to the bottom of the list
+                    if (newFocusedIndex < 0) {
+                        newFocusedIndex = this.props.listLength - 1;
+                    }
+
+                    return {focusedIndex: newFocusedIndex};
+                },
+                () => this.props.onFocusedIndexChanged(this.state.focusedIndex),
+            );
+        }, arrowUpConfig.descriptionKey, arrowUpConfig.modifiers, true);
+
+        this.unsubscribeArrowDownKey = KeyboardShortcut.subscribe(arrowDownConfig.shortcutKey, () => {
+            if (this.props.listLength <= 1) {
```

Comment:
@roryabraham edge case here - Focused index is on search, but pressing `enter` doesn't create a group. 

Changing it to `<=0` should fix it.

https://user-images.githubusercontent.com/29673073/155987443-8a9f6967-b935-4d80-97bf-05391f67b3db.mp4

Reply:
Sorry, I'm not able to reproduce this. Could you type up numbered reproduction steps?

Reply:
Sorry for not adding the steps. Also, my video is kinda pointless 😅

Steps

1. Create a new account, and login to the new account. (to have 0 contacts)
2. `Cmd + Shift + K` to open new group.
3. Search for a participant (eg: rushat@gmail.com)
4. Press `enter` while 

Expected result
The searched participant is selected.

Actual Result
Pressing `Enter` does nothing.

Reply:
My bad, this not a bug.